napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
Unfortunately it isn't available via pip and it seems library is abandoned (official information from the vendor). That is why I would like to make pip package. BTW I think I will rewrite module completely in the future because current one is prepared as cli tool not a library :)